book/src/prefabs/how_to_define_prefabs_adapter.md

```diff
@@ -84,16 +82,14 @@ If you are attempting to adapt a more complex type, please choose the appropriat
 
     ```rust,edition2018,no_run,noplaypen
     # extern crate amethyst;
-    # extern crate specs_derive;
-    # extern crate serde;
+        # extern crate serde;
```

Comment:
Could you align these lines you changed?

Reply:
Done